src/components/spreadsheet/table-wrapper.js

```diff
@@ -735,22 +679,111 @@ const TableWrapper = (props) => {
         ]
     );
 
+    const updateShuntCompensatorCells = useCallback((params) => {
```

Comment:
Could we move this function to a dedicated file ? I fear it will get out of control once there are multiple equipments handled this way